src/main/resources/application-localTest.properties

```diff
@@ -1,7 +1,7 @@
 # =========================================================================
 # Spring-related.
 spring.config.activate.on-profile=localTest
-spring.config.import=classpath:custom.properties
+spring.config.import=classpath:custom.properties, optional:file:${user.home}/.${user.name}-conf/uh-groupings-ui-overrides.properties
```

Comment:
I forgot to mention this file does not need the new properties file

Reply:
Done